src/components/App/App.js

```diff
@@ -23,6 +24,8 @@ function App() {
         exact path='/'
         component={ Home }
       />
+      <Route render= {() => <h1>Page Not found</h1>} />
```

Comment:
Glad you thought about what happens when a user tries to access a non-existent page!